frontend/public/scripts/products.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const getCookieValue = (cookieName) => {
+    const cookies = document.cookie.split("; ");
+    for (const cookie of cookies) {
+      const [key, value] = cookie.split("=");
+      if (key === cookieName) {
+        return value;
+      }
+    }
+    return null;
+  };
+
+  const accessToken = getCookieValue("accessToken");
+
+  if (!accessToken) {
+    window.open("login.html", "_self");
+    return;
+  }
+  const verifyToken = async () => {
```

Comment:
This function name does not match with what it is doing. What I see is fetchProducts or getProduct.